app/views/admin/products_v3/_table.html.haml

```diff
@@ -13,7 +13,7 @@
   = hidden_field_tag :producer_id, @producer_id
   = hidden_field_tag :category_id, @category_id
 
-  %table.products{ 'data-column-preferences-target': "table" }
+  %table.products{ 'data-column-preferences-target': "table", class: (hide_producer_column?(producer_options) ? 'hide-producer' : '') }
```

Comment:
I can't see a CSS rule for this class, did you forget to commit it?

Reply:
Hmm maybe it's not necessary at all: the column defaults take care of hiding it. 
So I guess we don't need this `hide_producer_column?` helper.

Reply:
> I can't see a CSS rule for this class, did you forget to commit it?

It's the existing style:
https://github.com/openfoodfoundation/openfoodnetwork/blob/68491559f351889864c7d15aa6a9c46631fd77e4/app/webpacker/css/admin/products_v3.scss#L185-L195

>  the column defaults take care of hiding it.

We need it because JS takes time to load and hide the column. Using it in the HTML immediately hides it when necessary.